qarysite/qary_app/serializers.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+from qary_app.models import Post, Chat, Document
+
+class ChatSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Chat
+        fields = '__all__'
```

Comment:
I think you want to put a new empty line (an <Enter> or '\n' char after '__all__') at the end of this file, just to comply with style